src/main/resources/application.yaml

```diff
@@ -144,7 +144,7 @@ envelope-access:
        updateService: bulk_scan_processor_tests
 
 reports:
-  jurisdictions: BULKSCAN, SSCS, PROBATE, DIVORCE
+  containers: bulkscan, sscs, probate, divorce
```

Comment:
Check if you use existing containers list defined in the application yaml else Pr looks good to go